README.rst

```diff
@@ -1,22 +1,25 @@
-``zope.component``
-==================
+.. image:: https://github.com/zopefoundation/zope.component/actions/workflows/tests.yml/badge.svg
+        :target: https://github.com/zopefoundation/zope.component/actions/workflows/tests.yml
+        :alt: Unit test status
+
+.. image:: https://coveralls.io/repos/github/zopefoundation/zope.component/badge.svg?branch=master
+        :target: https://coveralls.io/github/zopefoundation/zope.component?branch=master
+        :alt: Coverage Status
 
 .. image:: https://img.shields.io/pypi/v/zope.component.svg
-    :target: https://pypi.python.org/pypi/zope.component/
-    :alt: Latest Version
+        :target: https://pypi.python.org/pypi/zope.component/
+        :alt: Current version on PyPI
 
-.. image:: https://travis-ci.com/zopefoundation/zope.component.svg?branch=master
-        :target: https://travis-ci.com/zopefoundation/zope.component
-        :alt: Build Status
+.. image:: https://img.shields.io/pypi/pyversions/zope.component
+        :target: https://pypi.org/project/zope.component/
+        :alt: Supported Python versions
 
 .. image:: https://readthedocs.org/projects/zopecomponent/badge/?version=latest
         :target: http://zopecomponent.readthedocs.org/en/latest/
         :alt: Documentation Status
 
-.. image:: https://coveralls.io/repos/github/zopefoundation/zope.component/badge.svg?branch=master
-        :target: https://coveralls.io/github/zopefoundation/zope.component?branch=master
-        :alt: Coverage Status
-
+zope.component
+==============
```

Comment:
Why isn't the title marked up as a top-level header? This is a second-level header.

It was already broken that way, but since it's being edited here...

Reply:
All packages I have dealt with use this header level for the package title in the README

Reply:
And all packages I have dealt with use over/underlines for the package title (or I fix them to do so 😄 ).